samples/error-correction/syndrome/Syndrome.qs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+namespace Microsoft.Quantum.Samples.Hardware.Syndrome
```

Comment:
Given that this is no longer specific to hardware (though it should also work fine on hardware, of course), would it make sense to drop that from the namespace here?

Reply:
Sure, I'll change it to `ErrorCorrection`